### ISSUE TITLE
Issue loading mt3d model

### ISSUE THREAD

**Issue by paulinema** [modflowpy/flopy#244]
Hi,

I am new to Flopy and I am especially interested in post-processing results. I managed to load a flow model correctly but now I am trying to load a mt3d model (after a flow model) and I keep having these error messages:
"error loading name file entries from file:\n" + str(e))
Exception: error loading name file entries from file:
list index out of range

My code is simply:
import flopy
ws = r'T:\R230000\Python\K'
f1 = 'K_model_v6'
f2 = 'K_model_v9'
mf = flopy.modflow.Modflow.load(f1+'.mfn', exe_name='mf2005.exe', model_ws=ws)
mt = flopy.mt3d.mt.Mt3dms.load(f2+'.mts', version='mt3d-usgs',exe_name='mt3d-usgs.exe', modflowmodel=mf)
flopy.mt3d.mt.ftlfilename = f1+'.ftl'

I am using the native files which I have download from Modflow.

Anyone has an idea?
Thank you.

**Comment by langevin-usgs** [comment]
So just to clarify, the name of your mt3d name file is: 

    T:\R230000\Python\K\K_model_v9.mts

It would be helpful if you could show us the contents of that file.

And this line doesn't look right:

    flopy.mt3d.mt.ftlfilename = f1+'.ftl'

Instead you should add it to the mt constructor:

    mt = flopy.mt3d.mt.Mt3dms.load(f2+'.mts', version='mt3d-usgs',exe_name='mt3d-usgs.exe', 
    modflowmodel=mf, ftlfilename=f1+'.ftl')

Looks like it is something simple.

**Comment by paulinema** [comment]
Yes this is the right name of my file.

But when changing to your line mt = flopy.mt3d.mt.Mt3dms.load(f2+'.mts', version='mt3d-usgs',exe_name='mt3d-usgs.exe', modflowmodel=mf, ftlfilename=f1+'.ftl'), I get an error message saying: load() got an unexpected keyword argument 'ftlfilename'.

I tried an example where I created a flow and a mt3d models from scratch, then I tried to load them using the same syntax as in my first post, and it worked.

I have attached my mts file:
[K_model_v9.txt](https://github.com/modflowpy/flopy/files/1277828/K_model_v9.txt)

Thank you
